project.clj

```diff
@@ -18,7 +18,7 @@
 ;;   valid, and update the exclusions/comments accordingly
 ;; - Maybe you can just delete the dep! (doesn't hurt to check)
 
-(defproject ctia "1.0.0-rc1"
+(defproject ctia "1.1.0"
```

Comment:
You can't depend on a SNAPSHOT version while having a non SNAPSHOT version here.